R/get-path-from-object.R

```diff
@@ -253,5 +255,8 @@ find_nonmem_model_file_path <- function(.path, .check_exists = TRUE) {
   } else {
     res <- maybe_paths[which(exists_idx)]
   }
-  res
+
+  res %>%
```

Comment:
Hmmm, I changed it because some tests starting failing because of this when I moved the test files. It's because it was returning a path like `/user/whatever/tests/testthat/../../inst/test-refs/1.ctl` instead of `/user/whatever/inst/test-refs/1.ctl`, so some path comparison in the tests was failing.

I kinda lean toward leaving it like this because... a path like that _should_ be normalized in this way. For the record, this does _not_ mean it will return an absolute path. Example:

```
> p <- "inst/test-refs/1_param_table.R"
> normalizePath(p)
[1] "/data/home/sethg/rbabylon/inst/test-refs/1_param_table.R"
> as.character(fs::path_norm(p))
[1] "inst/test-refs/1_param_table.R"
```